corehq/apps/app_manager/suite_xml/sections/sync_requests.py

```diff
@@ -51,28 +61,8 @@ def get_module_contributions(self, module):
                     data=[
                         QueryData(
                             key='case_id',
-                            ref=(CaseTypeXpath(module.case_type)
-                                 .case(instance_name=RESULTS_INSTANCE)
-                                 .select(u'@status', u'open', quote=True)
-                                 .select_raw(0)
-                                 .slash(u'@case_id')),
-                            # e.g. instance('results')/results/case[@case_type='foo'][@status='open'][0]/@case_id
-                        ),
-                        QueryData(
-                            key='case_type',
-                            ref=(CaseTypeXpath(module.case_type)
-                                 .case(instance_name=RESULTS_INSTANCE)
-                                 .select(u'@status', u'open', quote=True)
-                                 .select_raw(0)
-                                 .slash(u'@case_type')),
-                        ),
-                        QueryData(
-                            key='case_name',
-                            ref=(CaseTypeXpath(module.case_type)
-                                 .case(instance_name=RESULTS_INSTANCE)
-                                 .select(u'@status', u'open', quote=True)
-                                 .select_raw(0)
-                                 .slash(u'name')),
+                            ref=SessionXPath('case_id'),
```

Comment:
That's not `commcaresession` though ... that's the session we instantiate in line 77, and populate in line 103 from the search results. If I understand correctly, it's independent of the module type. But even so, I think you make a good point that it's confusing at best, and maybe we should name it something other than "session", and its XPath class something less generic than `SessionXPath`.